src/blocker.cr

```diff
@@ -59,13 +59,22 @@ class Castblock::Blocker
         end
       end
 
-      if @mute_ads && (payload = message.payload_data) && payload.status.size > 0
-        if payload.status[0].custom_data.player_state == 1081 && payload.status[0].volume.muted == false
-          Log.info &.emit("Found ad, muting audio", device: device.name)
-          @chromecast.set_mute(device, true)
-        elsif payload.status[0].custom_data.player_state != 1081 && payload.status[0].volume.muted
-          Log.info &.emit("Ad ended, unmuting audio", device: device.name)
-          @chromecast.set_mute(device, false)
+      if (payload = message.payload_data) && payload.status.size > 0
+        if @mute_ads
+          # && (payload = message.payload_data) && payload.status.size > 0
+          if payload.status[0].custom_data.player_state == 1081 && payload.status[0].volume.muted == false
+            Log.info &.emit("Found ad, muting audio", device: device.name)
+           @chromecast.set_mute(device, true)
```

Comment:
I think the indent is a space off here, but otherwise it looks good! I got a few long ads and it worked almost every time. Occasionally it won't mute/skip because it never gets the broadcast message about the ad but that's a separate issue.

Reply:
Good catch! Fixed. 

Yeah I think I only noticed that once in the last week or so, but I haven't been paying 100% attention.